api/python/cell_census/README.md

```diff
@@ -1,13 +1,11 @@
-# CELLxGENE Cell Census
+# Cell Census of CZ CELLxGENE Discover
 
-The `cell_census` package provides an API to facilitate use of the CZI Science Cell Census. The Cell Census is a versioned container for the single-cell data hosted at [CELLxGENE Discover](https://cellxgene.cziscience.com/).
+The `cell_census` package provides an API to facilitate the use of the CZ CELLxGENE Discover Cell Census. For more information about the API and the project visit the [github repo](https://github.com/chanzuckerberg/cell-census/).
```

Comment:
tiny nit: it is best practice to actually display the repo name, as the contents may not be read in a browser.

Suggested change:
> visit the [chanzuckerberg/cell-census GitHub repo](https://github.com/chanzuckerberg/cell-census/)